gems/bootstrap/CVE-2018-14042.yml

```diff
@@ -0,0 +1,33 @@
+---
+gem: bootstrap
+cve: 2018-14042
+ghsa: 7mvr-5x2g-wfc8
+url: https://github.com/twbs/bootstrap/issues/26423
```

Comment:
I think the primary URL for this advisory should be `https://github.com/twbs/bootstrap/issues/26628`, which explains the vulnerability.

Reply:
fixed